testing/commontest.py

```diff
@@ -566,5 +566,22 @@ def iter_map(function, iterator):
         yield function(i)
 
 
+def xcopytree(source, dest, content=False):
+    """copytree can't copy all kind of files but is platform independent
+    hence we use it only if the 'cp' utility doesn't exist.
+    If content is True then dest is created if needed and
+    the content of the source is copied into dest and not source itself."""
+    if content:
+        subs = map(lambda d: os.path.join(source, d), os.listdir(source))
+        os.makedirs(dest, exist_ok=True)
+    else:
+        subs = (source,)
+    for sub in subs:
+        if shutil.which('cp'):
```

Comment:
Guessing it's mostly for windows ?

Reply:
Yep, but I didn't want to be specific, BeOS might not have `cp` either :-)